bessctl/conf/port/vhost/launch_container.py

```diff
@@ -115,14 +130,11 @@ def main(argv):
             time.sleep(100)
     except KeyboardInterrupt:
         pass
-    finally:
-        for proc in procs:
-            print('Terminating container (pid=%d)' % proc.pid)
-            proc.kill()
-            proc.wait()
 
-    return 0
+    for cid in range(num_containers):
```

Comment:
Minor: whether to have this loop in the `finally` block or not would exhibit slightly different behaviors, in case the script has not been interrupted by Ctrl+C but something else; the containers won't be terminated.

Reply:
Right